test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -24,16 +34,56 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "inline code with a line break" do
+      markdown = "The Game, `Mrs\nHudson`, is on!"
+      assert markdown == reformat(markdown)
```

Comment:
This test should probably fail, but it's included because there's complicated code inside of `earmark-parser` to track open code-blocks over multiple lines so changing the internal structure of the formatter could easily change the behavior.